### ISSUE TITLE
The story behind this package 

### ISSUE THREAD

**Issue by guyluz11** [janstol/yeedart#7]
As I am creating [smart Hub](https://github.com/CyBear-Jinni/CBJ_Hub) in dart I will definitely try this package and probably (if it is working) include it to add support for ‏Yeelight devices.

It will save me a lot of work so first of all many thanks.

Would love to hear why did you started this package.
What devices where tested.

If you know other dart packages to control smart devices products please let me know.

**Comment by janstol** [comment]
Well, at that time I was really into Dart/Flutter and was thinking what package I could create... I noticed Yeelight has a documentation for developers and I happened to have a Yeelight Lightstrip Plus (YLDD04YL) *(which is older model I think; bought in 2018)*. So I started experimenting with it and created this package.

I don't have any other Yeelight devices, so I can only test it on the lightstrip. 

I'm sorry for late response but I don't have much time/energy to maintain my packages regularly.
